01LessonOne/LessonOne.js

```diff
@@ -1,24 +1,28 @@
 module.exports = function() {
 
   this.returnTrue = function () {
-    // should return true
+    return true;
 
   }
 
   this.returnFalse = function() {
-    // should return false
+    return false;
 
   }
 
   this.greaterThan = function (big, small) {
     // should return true if the first argument is greater than the second argument
+    if(big>small) {return true;}
```

Comment:
What kind of value does the expression `big > small` result in? What are the possible values? What are the circumstances that lead to the expression being a particular value?

Also, for the sake of conforming to precedent coding styles, please indent your source like so:

```
if(big > small){
    return true;
}
else{
    return false;
}
```